### DISCUSSION TITLE
Failed in Fleet agent final_pipeline: field [created] not present as part of path [event.created]

### DISCUSSION THREAD

**Discussion by loganterfehr** [Security-Onion-Solutions/securityonion#12159] [Unsupported Versions]
I was looking through sysmon logs from my switches and I noticed that there each log has an error.message field that states this:

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/122332360/6e49b8ce-f4c5-49b7-8bb9-c1e3671a914d)

Is there a mismatch of logs being ingested? Or is this not a big deal. Thanks ahead of time!

**Comment by reyesj2** [comment]
The elasticagent integration cisco_ios https://github.com/Security-Onion-Solutions/securityonion/issues/12056 is included in the next release and will provide parsing for these logs.

Once the release is out and you upgrade you should be able to enable the integration and point your switch to send its logs to the port used by the integration (and update the firewall to also allow the integration port).
